app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsFragment.kt

```diff
@@ -188,7 +188,7 @@ class PaymentMethodsFragment : DaggerFragment(), PaymentMethodsView {
     var fiatPrice = "$fiatAmount $currency"
     if (isSubscription) {
       val period = Period.parse(frequency!!)
-      period?.mapToSubsCurrencyFrequency(context!!, fiatPrice)
+      period?.mapToSubsFrequency(context!!, fiatPrice)
```

Comment:
In the layout the payment methods list since it will only show a single payment method, the dialog should be adjusted to the correct size